Library/include/CSP/Multiplayer/SpaceEntity.h

```diff
@@ -206,6 +207,13 @@ class CSP_API SpaceEntity
 	/// @return SpaceEntitySystem
 	SpaceEntitySystem* GetSpaceEntitySystem();
 
+	void SetParentEntity(SpaceEntity* Parent);
+	void RemoveParentEntity();
+
+	SpaceEntity* GetParentEntity() const;
```

Comment:
Remember that all public API functions should be accompanied with doxygen-styled comments!